---
organizations:
  - name: Imperial College London
    url: https://www.imperial.ac.uk/dementia-research-institute/
  - name: UK Dementia Research Institute
    url: https://ukdri.ac.uk/
  - name: Department of Brain Sciences
    url: https://www.imperial.ac.uk/brain-sciences
superuser: false
authors:
  - admin
title: Neurogenomics Lab
role: ""
avatar_filename: avatar.jpg
bio: The UK Dementia Research Institute Neurogenomics Lab at Imperial College
#interests:
#  - ""
social:
  - icon_pack: ai
    icon: <i class="fab fa-github"></i>
    link: github.com/neurogenomics
  - icon: <i class="fab fa-docker"></i>
    icon_pack: ai
    link: https://hub.docker.com/orgs/neurogenomicslab
#education:
#  courses: []
email: ""
user_groups:
  - ""
---
Our research focuses on simple questions about complex brain traits: where do they occur (in which cell types)? When do they occur (at which developmental stages)? How do they occur (which quantitative traits of cells are affected)? We aim to develop the methods and datasets required to enable these questions to be answered systematically for any trait for which well-powered genetic data is available.

The most complete and unbiased data source currently available for the brain comes from single cell RNA-sequencing: an atlas of almost every cell type found in the central, peripheral and enteric nervous system is <a href="http://mousebrain.org/">available</a>. Using this dataset and other's like it we have investigated the cell types underlying <a href="https://www.nature.com/articles/s41588-018-0129-5.pdf">Schizophrenia</a>, <a href="https://www.biorxiv.org/content/10.1101/528463v1">Alzheimer's disease, Parkinsons, Stroke</a> and <a href="https://www.nature.com/articles/s41588-018-0152-6">cognitive ability</a>. To get greater accuracy in interpreting genetic data we need to go beyond mouse single cell RNA-seq data though: we plan to obtain human cell atlases using ATAC-seq and other genomic techniques.

Using transcriptomic datasets mapping expression changes across the lifespan we have investigated why the <a href="https://elifesciences.org/articles/17915">onset of psychiatric diseases</a> occurs in early adulthood (rather than in infancy, as would be expected for heavily genetic disorders). Currently this kind of data is only available for a small number of brain regions; we are looking to generate new datasets spanning the human lifespan for a range of disease relevant cell types. 

Identifying the causal cell types and developmental stages will not in itself lead to cures though. To get closer to that we need to understand what changes occur on the affected cells. Our approach is based on the notion that the causal changes associated with disease will be quantitative traits, much like height or BMI, but the important changes will be on the cellular level for specific subtypes of cells (i.e. increased spine density in Reln/Ndnf+ cortical interneurons).  To identify these changes and link them to genetics we need higher throughput biological methods. Towards this end we are building on the work of collaborators to develop <a href="https://www.biorxiv.org/content/early/2018/10/01/431957">spatial transcriptomic techniques</a> to enable simultaneous automated identification of all cell types found in tissue sections.

Looking for a postdoc, internship, masters or PhD? <a href="mailto:n.skene@imperial.ac.uk">Would be great to hear from you!</a>

Applications would be welcomed from both experimentalists and analysts. If you're background is not in biology but you're great at programming or mathematics then get in touch. Descriptions of potential projects will be put online soon: in the meanwhile, if this is what you want to work on then just get in touch.
